src/utils/productUtils/productOps.js

```diff
@@ -0,0 +1,13 @@
+import axios from "axios";
+
+export const getProducts = async (dispatch) => {
```

Comment:
here instead of taking `dispatch`, you can make a hook for naming `useAsync`.

When you will do like this, then you will not need to take dispatch as an argument. If you will face issue, then I can help you.